pysat/instruments/methods/madrigal.py

```diff
@@ -345,7 +345,7 @@ def get_remote_filenames(inst_code=None, kindat=None, user=None,
 
     # iterate over experiments to grab files for each one
     files = list()
-    print("Found {:d} Madrigral experiments".format(len(exp_list)))
+    print("Found {:d} Madrigal experiments".format(len(exp_list)))
```

Comment:
Out of scope for this PR, but should be a `logging.info` instead of a `print` statement, no?

Reply:
Probably.  We can change it here.